docs/reference/cluster/health.asciidoc

```diff
@@ -1,126 +1,168 @@
+    to false, which means it will not wait for relocating shards.
 
 `wait_for_nodes`::
-    The request waits until the specified number `N` of
+    (Optional, string) The request waits until the specified number `N` of
     nodes is available. It also accepts `>=N`, `<=N`, `>N` and `<N`.
     Alternatively, it is possible to use `ge(N)`, `le(N)`, `gt(N)` and
     `lt(N)` notation.
 
-`wait_for_events`::
-    Can be one of `immediate`, `urgent`, `high`, `normal`, `low`, `languid`.
-    Wait until all currently queued events with the given priority are processed.
+`wait_for_status`::
+    (Optional, string) One of `green`, `yellow` or `red`. Will wait (until the 
+    timeout provided) until the status of the cluster changes to the one
+    provided or better, i.e. `green` > `yellow` > `red`. By default, will not
+    wait for any status.
+
+[[cluster-health-api-response-body]]
+==== {api-response-body-title}
+
+`cluster_name`::
+    (string) The name of the cluster.
+
+`status`::
+    (string) The health status of the cluster.
+
+`timed_out`::
+    (boolean) If `false` the response returned within the period of 
+    time that is specified by the `timeout` parameter (`30s` by default).
+
+`number_of_nodes`::
+    (integer) The number of nodes within the cluster.
+
+`number_of_data_nodes`::
+    (integer) The number of nodes that are dedicated data nodes.
+
+`active_primary_shards`::
+    (integer) The number of active primary shards.
 
-`timeout`::
-    A time based parameter controlling how long to wait if one of
-    the wait_for_XXX are provided. Defaults to `30s`.
+`active_shards`::
+    (integer) The total number of active primary and replica shards.
 
-`master_timeout`::
-    A time based parameter controlling how long to wait if the master has not been
-    discovered yet or disconnected.
-    If not provided, uses the same value as `timeout`.
+`relocating_shards`::
+    (integer) The number of shards that are under relocation.
 
-`local`::
-    If `true` returns the local node information and does not provide
-    the state from master node. Default: `false`.
+`initializing_shards`::
+    (integer) The number of shards that are under initialization.
 
+`unassigned_shards`::
+    (integer) The number of shards that are not allocated.
+
+`delayed_unassigned_shards`::
+    (integer) The number of shards whose allocation has been delayed by the 
+    timeout settings.
+
+`number_of_pending_tasks`::
+    (integer) The number of cluster-level changes that have not yet been 
+    executed.
+
+`number_of_in_flight_fetch`::
+    (integer) The number of unfinished fetches.
+
+`task_max_waiting_in_queue_millis`::
+    (integer) The time expressed in milliseconds since the earliest initiated task 
+    is waiting for being performed.
+
+`active_shards_percent_as_number`::
+    (float) The ratio of active shards in the cluster expressed as a percentage. 
+
+[[cluster-health-api-example]]
+==== {api-examples-title}
+
```

Comment:
Adding this request should clear up the elasticsearch-ci/docs-check and elasticsearch-ci/oss-distro-docs tests.
```suggestion

[source,js]
--------------------------------------------------
GET _cluster/health
--------------------------------------------------
// CONSOLE
// TEST[s/^/PUT test1\n/]

```